### PR TITLE
update ui components to have 1 version call

### PR DESCRIPTION
**Pull request by willkutler**
https://github.com/open-cluster-management/backlog/issues/10974

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -119,13 +119,13 @@ module.exports = {
         test: /\.css$/,
         include: [
           path.resolve(__dirname, './node_modules/monaco-editor'),
-          path.resolve(__dirname, './node_modules/@patternfly'),
+          path.resolve(__dirname, './node_modules/@open-cluster-management/ui-components'),
```
Comment (ycao56) [inline webpack.config.js:122]: Any reason why this change is needed?

Comment (willkutler) [reply webpack.config.js:122]: @ycao56 the style got messed up when i updated ui-components. I assumed there was a style change in that repo, and it shouldn't change anything in grc since ui-components should include the patternfly styles

Comment (ycao56) [reply webpack.config.js:122]: ui-components does not import any styles. that's why we include `@patternfly` styles explicitly and skip any styles from `@open-cluster-management/ui-components`
